### ISSUE TITLE
iamlive crashes when AWS GovCloud resources involved

### ISSUE THREAD

**Issue by BillKlineVT** [iann0036/iamlive#34]
When using iamlive in conjunction with AWS GovCloud CLI/SDK calls in proxy mode, iamlive crashes when parsing the [GovCloud global endpoints](https://aws.amazon.com/compliance/fips/) that have the region of "us-gov" (example: IAM, Route53)

```
Error:
panic: runtime error: slice bounds out of range [:7] with length 6
github.com/iann0036/iamlive/iamlivecore.subARNParameters(<sanitized>)
/Users/runner/work/iamlive/iamlive/iamlivecore/logger.go:699
```

Looking at https://github.com/iann0036/iamlive/blob/main/iamlivecore/logger.go#L699, it appears in the case of a global endpoint, the Region struct field is 6 chars (`us-gov`) and the string slice on Line 699 is expecting 7 chars.  Note that this works for non-global GovCloud ARNs since they are region specific, hence start with `us-gov-<region>`.

It looks like all that needs to be done is to update Line 699 to the following:
`if call.Region[0:6] == "us-gov" {`
and this would apply to both global `us-gov` and non-global `us-gov-<region>` references.

Forgive my lack of experience with Go syntax so I could be wrong here, but it looks to be the culprit.

**Comment by iann0036** [comment]
Hey @BillKlineVT,

Thanks heaps for raising! This should be fixed in v0.38.0, but let me know if otherwise.

**Comment by BillKlineVT** [comment]
Works great in v0.38.0, thank you for fixing so quickly!
